### PR TITLE
feat(Storage): Combine support for all APIs

### PR DESCRIPTION
**Pull request by palpatim**
- Updated README with Predictions & Storage

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: AmplifyCombineSupport/AmplifyCombineSupportTests/GeneralTests.swift [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+//
+
+import XCTest
+
+import AmplifyCombineSupport
+
+@testable import Amplify
+@testable import AmplifyTestCommon
+
+class StorageGeneralTests: XCTestCase {
```
Comment (lawmicha) [inline AmplifyCombineSupport/AmplifyCombineSupportTests/GeneralTests.swift:15]: file name GeneralTest.swift -> StorageGeneralTests.swift, and it is missing from AmplifyCombineSupportTests/Storage. opening the folder in finder shows the file though

File: AmplifyCombineSupport/AmplifyCombineSupportTests/StorageCombineSupportChainTests.swift [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+//
+
+import XCTest
+
+import AmplifyCombineSupport
+
+@testable import Amplify
+@testable import AmplifyTestCommon
+
+class ChainTests: XCTestCase {
```
Comment (lawmicha) [inline AmplifyCombineSupport/AmplifyCombineSupportTests/StorageCombineSupportChainTests.swift:15]: `class StorageCombineSupportChainTests` and also don't see this file added to target

File: AmplifyCombineSupport/AmplifyCombineSupportTests/GeneralTests.swift (hunk 1 above)
Comment (palpatim) [reply AmplifyCombineSupport/AmplifyCombineSupportTests/GeneralTests.swift:15]: Grr. Had some problems with Xcode losing file references when renaming. Will fix.

File: AmplifyCombineSupport/AmplifyCombineSupportTests/StorageCombineSupportChainTests.swift (hunk 2 above)
Comment (palpatim) [reply AmplifyCombineSupport/AmplifyCombineSupportTests/StorageCombineSupportChainTests.swift:15]: Same problem w/file renaming. Removed

**Pull request merged**